docker/production/scripts/init-db.sh

```
@@ -0,0 +1,145 @@
+    echo "  (no drugref seed data found, skipping)"
+fi
+
+# -----------------------------------------------------------------------------
+# Step 2: Apply migration scripts
+# -----------------------------------------------------------------------------
+# Migration order matters. List mirrors .devcontainer/db/scripts/populate_db.sh
+# plus all migrations released since. Keep in chronological order.
+
+echo "[3/4] Applying schema migrations..."
+apply_update() {
+    local file="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)